src/scenes/CreateEvent/LanguageSelector.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { observer, inject } from 'mobx-react';
+import { defineMessages } from 'react-intl';
+import { Select, MenuItem } from '@material-ui/core';
+
+import { Section } from './components';
+
+const messages = defineMessages({
+  languagSelector: {
+    id: 'create.languagSelector',
+    defaultMessage: 'Language',
+  },
+});
+
+const LanguageSelector = observer(({ store: { createEvent } }) => (
+  <Section title={messages.languagSelector}>
+    <Select
+      value={createEvent.language}
+      onChange={(e) => createEvent.language = e.target.value}
+      name="lang"
+    >
+      <MenuItem value="English">English</MenuItem>
+      <MenuItem value="Chinese">中文</MenuItem>
+      <MenuItem value="Korean">한국어</MenuItem>
+    </Select>
+  </Section>
+));
+
+
```

Comment:
👾

Reply:
Hi Deric I'm wondering what does this emoji mean, would you mind to tell me?

Reply:
@ippudo lol its a space invader. before your time. it means remove the extra line.

https://en.wikipedia.org/wiki/Space_Invaders